backend/src/core/constants.ts

```diff
@@ -20,6 +20,12 @@ export enum MessageStatus {
   InvalidRecipient = 'INVALID_RECIPIENT',
 }
 
+export enum DefaultCredentialName {
+  Email = 'EMAIL_DEFAULT',
+  SMS = 'SMS_DEFAULT',
```

Comment:
Perhaps we want to read the default credential names for SMS/Telegram from config so that we can have the option of using different creds for each deployment environments (staging, production).

Reply:
yep setting it up now demo/staging/SMS_DEFAULT

Reply:
```  if (label === DefaultCredentialName.SMS) {
      const campaign = await SmsService.findCampaign(+campaignId, userId) // TODO: refactor this into res.locals
      if (campaign.demoMessageLimit) {
        credentialName = `demo/${config.get('env')}/${label}`
      } else {```